lib/util/Helpers.ts

```diff
@@ -1,41 +1,118 @@
+	return typeof data !== 'string' ? format('%O', data) : data;
+}
+
+/**
+ * Factory function to create painter for log output coloring.
  *
- * @param   arr    The string array to check.
- * @param   value  Value to check the existance.
- * @returns Wheter or not the string is in the provided array.
+ * @returns Readonly object with level tags as painter methods.
  */
-export function includes(arr: string[], value: string): boolean {
-	return arr.indexOf(value) > -1;
+export function colorized(): Readonly<LoggerColorized> {
+	const output = (c: keyof typeof Colors): ((text: string) => string) =>
+		color()
+			? (text: string) => Colors[c] + text + '\x1b[0m'
+			: (text: string) => text;
+
+	return Object.freeze({
+		SILLY: output('magenta') /** @deprecated */,
+		DEBUG: output('magenta'),
+		AUDIT: output('blue'),
+		HTTP: output('green') /** @deprecated */,
+		INFO: output('green'),
+		WARNING: output('yellow'),
+		ERROR: output('red'),
+		CRITICAL: output('red') /** @deprecated */
+	});
+}
+
+/**
+ * Function for checking if data is a key/value pair object.
+ *
+ * @param   data  The data being checked.
+ * @returns Whether or not the data is an object.
+ */
+export function isJsonObject(data: unknown): boolean {
+	return typeof data === 'object' && !Array.isArray(data) && data !== null;
+}
+
+/**
+ * Function for checking if data is an array object.
+ *
+ * @param   data  The data being checked.
+ * @returns Whether or not the data is an array.
+ */
+export function isJsonArray(data: unknown): boolean {
+	return typeof data === 'object' && Array.isArray(data) && data !== null;
+}
+
+/**
+ * Retrieve the current log level name.
+ *
+ * @returns The level name.
+ */
+export function level(): keyof typeof LogLevels {
+	const input = (process.env.OZLOGGER_LEVEL?.toLowerCase() ||
+		'debug') as keyof typeof LogLevels;
+
+	return input in LogLevels ? input : 'audit';
 }
 
 /**
  * Check if the log outputs should be colorized.
  *
- * @returns Wheter or not the output can be colorized.
+ * @returns Whether or not the output can be colorized.
  */
 export function color(): boolean {
 	return !!process.env.OZLOGGER_COLORS?.match(/true/i);
 }
 
 /**
- * Outputs the stringified version of the input data.
+ * Check which output should be used for logging.
  *
- * @param   data  The data to stringify.
- * @returns The stringified data.
+ * @returns Output name to be used.
  */
-export function stringify(data: unknown): string {
-	return typeof data !== 'string' ? format('%O', data) : data;
+export function output(): (typeof Outputs)[number] {
+	const input = (process.env.OZLOGGER_OUTPUT?.toLowerCase() ||
+		'json') as (typeof Outputs)[number];
+
+	return Outputs.indexOf(input) < 0 ? 'json' : input;
+}
+
+/**
+ * Parse the log server port and interface to bind to.
+ *
+ * @returns The port number and interface address.
+ */
+export function host(): [number, string] {
+	// @todo Leandro: Implement support for the following input formats:
```

Comment:
That's just for future reference, for now I'm not planning on implementing it :sweat_smile: 
However I think it would give more flexibility for different deployment scenarios